docs/understanding-tracking-design/defining-the-data-to-collect-with-data-poducts/api/index.md

```diff
@@ -0,0 +1,144 @@
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}`
+
+Query parameters `organizationId` and `dataProductId` are required.
+
+### Retrieve History Information for a Data Product
+
+If you wish to retrieve information about a specific data product, you can use the following GET request:
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}/history`
+
+You can pass several parameters to control the result of the response:
+
+- **before**: returns records equal or less than the timestamp in the ISO-8601 format
+- **limit**: limits the number of records
+- **offset**: skip the first N results
+- **order**: order of returned records, `asc` or `desc`. Defaults to `desc`.
+
+Query parameter `organizationId` is required.
+
+## Creating and updating Data Products
+
+### Creating a Data Product
+
+This `POST` request allows you to create a new data product within an organization.
+
+`**POST** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1`
+
+The request body is mandatory and should be in JSON format. Here’s an example of how the request body might look:
+
+```json
+{
+  "name": "Performance tracking",
+  "description": "Tracks performance",
+  "domain": "Marketing",
+  "owner": "IT department",
+  "accessInstructions": "The data can be accessed through company Superset"
+}
+```
+
+### Updating a Data Product
+
+Use this request to update a data product. The `dataProductId` is required, along with a valid request body.
+
+`**POST** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}`
+
+See the [detailed API documentation](https://console.snowplowanalytics.com/api/msc/v1/docs) for all options.
+
+### Delete a Data Product
+
+Use this request to delete a data product. The `dataProductId` and `organizationId` are both required.
+
+`**POST** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}`
+
+## Subscription Management for Data Products
+
+### Retrieve All Subscriptions for a Data Product
+
+To retrieve all subscriptions for a data product, use the following request. The `organizationId` and `dataProductId` are required.
+
+`**GET** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}/subscriptions`
+
+### Add a Subscription
+
+To add a subscription for a data product, use the following request. The `organizationId`, `dataProductId` and a valid request body are required.
+
+`**POST** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}/subscriptions`
+
+### Update a Subscription
+
+To update a subscription for a specific data product, use the following request. The `organizationId`, `subscriptionId`, `dataProductId`, and a valid request body are required.
+
+`**PUT** ​/api​/msc​/v1​/organizations/{organizationId}/data-products/v1/{dataProductId}/subscriptions/{subscriptionId}`
+
+### Delete a Subscription
```

Comment:
This endpoint can be though of as the action of `unsubscribe`.

Reply:
done